### PR TITLE
 toString functionality for temporal and duration

### PR DESCRIPTION
**Pull request by Lojjs**

### CODE REVIEW

File: community/values/src/main/java/org/neo4j/values/storable/TemporalValue.java [hunk 1]
```diff
@@ -172,12 +172,6 @@ public final NumberType numberType()
         return NO_NUMBER;
     }
 
-    @Override
-    public final String toString()
-    {
-        return getClass().getSimpleName() + "<" + prettyPrint() + ">";
```
Comment (sherfert) [inline community/values/src/main/java/org/neo4j/values/storable/TemporalValue.java:182]: If the implementation is always `prettyPrint()`, why not just change it here in the superclass?

File: enterprise/cypher/spec-suite-tools/src/test/scala/cypher/features/Neo4jExceptionToExecutionFailed.scala [hunk 2]
```diff
@@ -32,8 +32,8 @@ object Neo4jExceptionToExecutionFailed {
 
   def convert(phase: String, t: Throwable): ExecutionFailed = {
     val neo4jException = t match {
-      case re: RuntimeException => re
```
Comment (sherfert) [inline enterprise/cypher/spec-suite-tools/src/test/scala/cypher/features/Neo4jExceptionToExecutionFailed.scala]: That should not have been comitted.

File: community/values/src/main/java/org/neo4j/values/storable/TemporalValue.java (hunk 1 above)
Comment (Lojjs) [reply community/values/src/main/java/org/neo4j/values/storable/TemporalValue.java:182]: That is a good point, have changed it now

File: enterprise/cypher/spec-suite-tools/src/test/scala/cypher/features/Neo4jExceptionToExecutionFailed.scala (hunk 2 above)
Comment (Lojjs) [reply enterprise/cypher/spec-suite-tools/src/test/scala/cypher/features/Neo4jExceptionToExecutionFailed.scala]: Oops

**Review by craigtaverner — APPROVED** [review]

**Pull request merged**